### PR TITLE
DV Filter on additional reports

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/models/concerns/reporting/monthly_reports/monthly_report_charts.rb [hunk 1]
```diff
@@ -326,6 +341,7 @@ def homeless_project_types
     end
 
     def census_by_project_type
+      # raise 'hi'
```
Comment (awisema) [inline app/models/concerns/reporting/monthly_reports/monthly_report_charts.rb]:
```suggestion
```

File: app/models/concerns/reporting/monthly_reports/monthly_report_charts.rb [hunk 2]
```diff
@@ -193,6 +194,20 @@ def self.filter_for_coc_codes(coc_codes)
...
 
+    def self.filter_for_dv_status(filter)
+      return current_scope if filter.dv_status.blank?
+
+      client_ids = GrdaWarehouse::ServiceHistoryEnrollment.entry.joins(:project, enrollment: :health_and_dvs).
+        merge(GrdaWarehouse::Hud::Project.where(id: filter.anded_effective_project_ids)).
+        merge(
+          GrdaWarehouse::Hud::HealthAndDv.where(
+            InformationDate: filter.range,
+            DomesticViolenceVictim: filter.dv_status,
```
Comment (awisema) [inline app/models/concerns/reporting/monthly_reports/monthly_report_charts.rb]: Flag this so we remember to change it for FY2024?

**Review by awisema — APPROVED** [review]

File: app/models/concerns/reporting/monthly_reports/monthly_report_charts.rb (hunk 2 above)
Comment (eanders) [reply app/models/concerns/reporting/monthly_reports/monthly_report_charts.rb]: Thanks!

File: app/models/concerns/reporting/monthly_reports/monthly_report_charts.rb (hunk 1 above)
Comment (eanders) [reply app/models/concerns/reporting/monthly_reports/monthly_report_charts.rb]: Whoops!

**Pull request merged**